File: Modul-3/Soal-2/PRAK302-2410817210021-MuhammadNafisPutra.c
#include <stdio.h>

int main(){
    int nilai;
    char Huruf;

    printf("Masukkan Nilai: ");
    scanf("%d", &nilai);

    if (nilai >= 80) {
        Huruf = 'A';
    } else if (nilai >= 70) {
        Huruf = 'B';
    } else if (nilai >= 60) {
        Huruf = 'C';
    } else if (nilai >= 50) {
        Huruf = 'D';
    } else {
        Huruf ='E';
    }
    printf("%d : %c\n", nilai, Huruf);

    return 0;
    }

File: Modul-3/Soal-2/PRAK302-2410817210021-MuhammadNafisPutra.py
nilai = int (input("Masukkan Nilai: "))

if nilai >= 80:
    Huruf = 'A'
elif nilai >= 70:
    Huruf = 'B'
elif nilai >= 60:
    Huruf = 'C'
elif nilai >= 50:
    Huruf = 'D'
else:
    Huruf = 'E'
print(f"{nilai} : {Huruf}")